plugin.yml

```diff
@@ -1,9 +1,89 @@
 name: AFKPGC
 main: com.github.Kraken3.AFKPGC.AFKPGC
-author: Kraken3
-version: 1.2.4
+author: Kraken3/Maxopoly

```

Comment:
I think convention is to add an "authors" line for major contributors that weren't the original author. Just a way to give props to the originator but still get props for your hard work.

For an example, see [Humbug](https://github.com/Civcraft/Humbug/blob/master/plugin.yml)